### PR TITLE
Add chinese BERT config

### PR DESCRIPTION
**Pull request by PaGul**
Add a configuration for the Chinese SQuAD that uses Chinese BERT.
Trained on DRCD (https://arxiv.org/abs/1806.00920)

### CODE REVIEW

File: docs/features/models/squad.rst [hunk 1]
```diff
@@ -218,3 +218,26 @@ Link to SDSJ Task B dataset: http://files.deeppavlov.ai/datasets/sber_squad-v1.1
...
+	
+Link to DRCD dataset: http://files.deeppavlov.ai/datasets/DRCD.tar.gz
+Link to DRCD paper: https://arxiv.org/abs/1806.00920
+
++------------------------------------------------------------------------+----------------+-----------------+
+| Model config                                                           |    EM (dev)    |    F-1 (dev)    |
++========================================================================+================+=================+
+| :config:`DeepPavlov ChineseBERT <squad/squad_zh_zh_bert.json>`         |   85.13+-0.26  |    85.15+-0.12  |
++------------------------------------------------------------------------+----------------+-----------------+
+| :config:`DeepPavlov multilingual BERT <squad/squad_zh_bert.json>`.     |   84.18+-0.34  |    84.08+-0.07  |
```
Comment (yoptar) [inline docs/features/models/squad.rst]: Naming bothers me. Maybe `squad_zh_bert_mult` and `squad_zh_bert_zh`? What do you think, @yurakuratov?

Comment (PaGul) [reply docs/features/models/squad.rst]: Added tests with as yet old names

Comment (yurakuratov) [reply docs/features/models/squad.rst]: Yeah, I agree that current names are not the best choice. `squad_zh_bert_mult` and `squad_zh_bert_zh` make more sense.

Comment (PaGul) [reply docs/features/models/squad.rst]: Done

**Comment by PaGul** [comment]
Done

**Pull request merged**